File: src/main/java/com/sica/backend/entity/Area.java
package com.sica.backend.entity;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Table(name = "areas")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Area {

    @Id
    @GeneratedValue(strategy = GenerationType.UUID)
    private String id;

    @Column(nullable = false)
    private String name;

    private String description;

    @Column(nullable = false)
    private Boolean active = true;
}

File: src/main/kotlin/com/sica/backend/entity/Area.kt
package com.sica.backend.entity

import jakarta.persistence.*

@Entity
@Table(name = "areas")
data class Area(
    @Id
    @GeneratedValue(strategy = GenerationType.UUID)
    val id: String? = null,

    @Column(nullable = false)
    val name: String,

    val description: String? = null,

    @Column(nullable = false)
    var active: Boolean = true
)
